numpy/polynomial/_polybase.py

```diff
@@ -123,6 +123,10 @@ def _line(self):
         pass
 
     @abstractmethod
+    def _interp(self):
+        pass
```

Comment:
Ah, because of what you said before, where abstractmethods are required to be implemented by all subclasses. Ok, what are the options for something like this though? This will simply require an alert to anyone who relies on ABCPolyBase to update their implementations accordingly. Is that logistically difficult?